example/src/App.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
```

Comment:
Looks like `prettier` isn't configured to format the example directory. Normally, we'd see single quotes in Garden.

Reply:
I was using the default prettier and eslint config within `create-react-app` in the beginning.

I've now brought over the standard Garden prettier configuration and our react eslint config. I've also updated husky to run the `lint` and `format` commands across all packages for it's git hooks.